--- a/libcinnamon-desktop/cdesktop-enums.h	2017-01-14 08:50:19.583860736 +0000
+++ b/libcinnamon-desktop/cdesktop-enums.h	2017-01-14 08:51:17.323858486 +0000
@@ -200,7 +200,6 @@
         C_DESKTOP_MEDIA_KEY_WWW,
         C_DESKTOP_MEDIA_KEY_VIDEO_OUT,
         C_DESKTOP_MEDIA_KEY_ROTATE_VIDEO,
-        C_DESKTOP_MEDIA_KEY_ROTATE_VIDEO_LOCK,
         C_DESKTOP_MEDIA_KEY_SCREENREADER,
         C_DESKTOP_MEDIA_KEY_ON_SCREEN_KEYBOARD,
         C_DESKTOP_MEDIA_KEY_INCREASE_TEXT,
@@ -261,7 +260,6 @@
         [C_DESKTOP_MEDIA_KEY_WWW] = "www",
         [C_DESKTOP_MEDIA_KEY_VIDEO_OUT] = "video-outputs",
         [C_DESKTOP_MEDIA_KEY_ROTATE_VIDEO] = "video-rotation",
-        [C_DESKTOP_MEDIA_KEY_ROTATE_VIDEO_LOCK] = "video-rotation-lock",
         [C_DESKTOP_MEDIA_KEY_SCREENREADER] = "screenreader",
         [C_DESKTOP_MEDIA_KEY_ON_SCREEN_KEYBOARD] = "on-screen-keyboard",
         [C_DESKTOP_MEDIA_KEY_INCREASE_TEXT] = "increase-text-size",
